webpack.config.ts

```diff
@@ -79,6 +80,16 @@ const config: Configuration = {
 	},
 	module: {
 		rules: [
+			{
+				test: /octicons-react\//,
+				loader: 'string-replace-loader',
+				options: {
+					search: /\n(.*)Icon.defaultProps = {\n\s+className: octicon/g,
+					replace: (match: string, name: string) => {
+						return match.replace('octicon', 'octicon octicon-' + decamelize(name, '-'));
```

Comment:
Maybe add a comment with a link to https://github.com/primer/octicons/pull/453?

Reply:
We’re all notified when that PR closes. If merged one of us will drop the code. If rejected the comment will become unnecessary.